sys-devel/dwz/dwz-0.13.ebuild

```
@@ -5,11 +5,11 @@ EAPI=7
 
 inherit toolchain-funcs
 
-FPKG_HASH="1624afa75b94633e03c6e1bb952fb348"
-
 DESCRIPTION="DWARF optimization and duplicate removal tool"
-HOMEPAGE="https://sourceware.org/git/?p=dwz.git;a=summary"
-SRC_URI="https://src.fedoraproject.org/repo/pkgs/dwz/${P}.tar.bz2/${FPKG_HASH}/${P}.tar.bz2"
+HOMEPAGE="https://sourceware.org/dwz"
+SRC_URI="https://sourceware.org/ftp/dwz/releases/dwz-0.13.tar.xz"
+
+S="${WORKDIR}/${PN}"
 
 LICENSE="GPL-2+ GPL-3+"
```

From looking at the COPYING file and the headers in the source files, I think this should just be `GPL-2+`.

```suggestion
LICENSE="GPL-2+"
```

---

It also has COPYING3 file, and at least dwarf2.h and sha1.h use GPL3+ licensing

---

Sorry, I missed that. Thank you for double checking :)